<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterForm</class>
 <widget class="QWidget" name="RegisterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>330</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Palatino Linotype</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Zemini</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../My Pictures/groovie_logo.png</normaloff>../../../My Pictures/groovie_logo.png</iconset>
  </property>
  <widget class="QLabel" name="lb_welcome">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Welcome!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_next">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>471</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>User</string>
    </property>
    <widget class="QLabel" name="lb_mail">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Email: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_password">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_username">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_mail">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_password">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lb_username">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lb_user_picture">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>70</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_sign_up">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Sign up&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_zemini_terms">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>By clicking on &quot;OK&quot;, you agree that you accept  our &lt;br/&gt; &lt;a href=&quot;www.groovieandroid.com&quot;&gt;terms  of service&lt;/a&gt; 
  and our &lt;a href=&quot;www.groovieandroid.com&quot;&gt;privacy policy&lt;/a&gt; </string>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_error_text">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_sign_in">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;a href=&quot;#&quot;&gt;Sign in&lt;/a&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
